sequencer/api/migrations/postgres/V501__epoch_tables.sql

```diff
@@ -51,4 +42,4 @@ CREATE TABLE epoch_migration (
     completed bool DEFAULT FALSE
 );
 
-INSERT INTO epoch_migration (table_name) VALUES ('anchor_leaf'), ('da_proposal'), ('vid_share'), ('undecided_state'), ('quorum_proposals'), ('quorum_certificate');
\ No newline at end of file
+INSERT INTO epoch_migration (table_name) VALUES ('anchor_leaf'), ('da_proposal'), ('vid_share'), ('quorum_proposals'), ('quorum_certificate');
```

Comment:
I don't know what this is exactly but I guess we want something like `DELETE FROM epoch_migration WHERE table_name = 'undecided_state'`?